libsource/src/main/java/com/virtusize/libsource/network/VirtusizeApi.kt

```diff
@@ -83,14 +83,14 @@ internal object VirtusizeApi {
      * @param product the VirtusizeProduct for which Fit Illustrator URL is needed
      * @return the Fit Illustrator URL as String
      */
-    fun fitIllustrator(product: VirtusizeProduct): String {
+    fun fitIllustrator(product: VirtusizeProduct, randomNumberString: String? = null): String {
```

Comment:
I think here we should do the same that iOS, that `randomNumber` is just an implementation detail.

Reply:
ok, it's changed